package.json

```diff
@@ -2,21 +2,33 @@
     "devDependencies": {
         "@babel/runtime": "^7.11.0",
         "ajv": "^6.10.2",
+        "babel-eslint": "^10.1.0",
         "bent": "^7.3.9",
         "chunk": "^0.0.2",
+        "countries-list": "^2.6.1",
+        "eslint": "^7.32.0",
+        "eslint-plugin-json": "^3.1.0",
+        "eslint-plugin-optimize-regex": "^1.2.1",
         "glob": "^7.1.6",
         "husky": "^3.1.0",
-        "typesense": "^0.6.1",
-        "countries-list": "^2.6.1"
+        "lint-staged": "^11.1.2",
+        "prettier": "^2.3.2",
+        "typesense": "^0.6.1"
     },
     "scripts": {
-        "test": "node test-records.js",
+        "test": "lint-staged && node test-records.js # this order allows useful hints for invalid files",
```

Comment:
You're running ESLint twice in CI and `yarn test` shouldn't call `prettier --write`. Here's what you want:

* Have `test` as `eslint && test-records.js`.
* In `lint-staged`, run `yarn test` instead of ESLint.
* Revert the changes in the CI config.